lib/web_ui/lib/src/engine/canvaskit/canvaskit_api.dart

```diff
@@ -96,6 +96,9 @@ class CanvasKit {
   );
   external SkSurface MakeSWCanvasSurface(html.CanvasElement canvas);
   external void setCurrentContext(int glContext);
+
+  /// Creates an [SkPath] using commands obtained from [SkPath.toCmds].
+  external SkPath MakePathFromCmds(List<dynamic> pathCommands);
```

Comment:
Use `SkPath.MakeFromCmds` instead. This is deprecated and I think it's removed in the newest CanvasKit

Reply:
Added TODO to switch in the upcoming version of CanvasKit.